index.js

```diff
@@ -68,8 +104,7 @@ app.post('/register', function(req, res) {
       key: req.body.key,
       name: req.body.name,
       active: true
-    })
-    .then(function() {
+    }).then(function() {
```

Comment:
Yes - it was just I realized we've got a convention of putting .then in the next line ...